spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -353,7 +353,11 @@ def main(argv: Sequence[str]):
         printv('\nApply straightening to disc labels...', verbose)
         sct_apply_transfo.main(['-i', fname_disc, '-d', 'data_straightr.nii', '-w', 'warp_curve2straight.nii.gz',
                                 '-o', 'labeldisc_straight.nii.gz', '-x', 'label', '-v', '0'])
-        label_vert('segmentation_straight.nii', 'labeldisc_straight.nii.gz')
+        try:
+            label_vert('segmentation_straight.nii', 'labeldisc_straight.nii.gz')
+        except ValueError:
+            printv(f"No disc labels found in straightened version of discfile {fname_disc}", 1, 'error')
+            sys.exit(1)
```

Comment:
This is a nice example of what you meant on https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4199 re: catching the API exception, then going with `printv` + `sys.exit` instead. I like it! :)

Reply:
The only caveat with this pattern is the case where multiple different `ValueError`s could be thrown? And, the `except:` masks the true cause of the exception?

So, there's maybe a motivation to check the error message itself, or create more specific error classes for our case, as per [this thread](https://stackoverflow.com/a/13531310). But, honestly, that might just be complicating things unnecessarily, given that this is a simple fix.